tests/unit/test-util.js

```diff
@@ -86,15 +97,23 @@ const testSingleResource = (serializedResource, resourceType, resourceId, nested
  * @param {object} rawResources Raw resources to be used in test.
  * @param {*} resourceType The type of the resource.
  * @param {*} resourceKey The id of the resource.
+ * @param {object} relationships The relationships of the resource.
  */
-const testMultipleResources = (serializedResources, rawResources, resourceType, resourceKey) => {
+const testMultipleResources = (serializedResources,
+  rawResources, resourceType, resourceKey, relationships) => {
```

Comment:
Could you fix the indentation here? https://github.com/airbnb/javascript#functions--signature-invocation-indentation